--- Makefile	2015-08-24 16:09:29.170152630 +0200
+++ Makefile_f	2015-08-24 16:25:13.856183381 +0200
@@ -85,7 +85,7 @@
 #######################################
 # macros for gcc
 AS_DEFS =
-C_DEFS = -D__weak=__attribute__((weak)) -D__packed=__attribute__((__packed__)) -DUSE_HAL_DRIVER -DSTM32F303xC
+C_DEFS = -D__weak="__attribute__((weak))" -D__packed="__attribute__((__packed__))" -DUSE_HAL_DRIVER -DSTM32F303xC
 # includes for gcc
 AS_INCLUDES =
 C_INCLUDES = -IInc
